### ISSUE TITLE
What is the accuracy on SQuAD?

### ISSUE THREAD

**Issue by Hrant-Khachatrian** [hsgodhia/squad_rasor_nn#1]
Did you manage to get close to the numbers reported in the paper?

**Comment by hsgodhia** [comment]
Hi @Hrant-Khachatrian we didn't get the same numbers but are pretty sure it is some bug in our code. Incase you write a Pytorch version would be great to see the code

**Comment by Hrant-Khachatrian** [comment]
Thanks for the reply!

We try to implement it in [Tensorflow](https://github.com/YerevaNN/RaSoR-in-Tensorflow) and can't reproduce the results either. We'll try to compare our implementation with your code :)
